googletest/test/gtest_setuptestcase_failure_test.cc

```diff
@@ -0,0 +1,58 @@
+// Copyright 2009 Google Inc.  All rights reserved.
```

Comment:
What is the reason to remove this test from gtest_unittest and make is standalone? There are multiple changes needed to support it. Not clear what benefit it provides? Thank you

Reply:
1. FAIL on AdHocTestResultTest::SetUpTestCase
1. expect RUN_ALL_TESTS return 1
1. gtest_unittest expect RUN_ALL_TESTS return 0

If there is another test that expects RUN_ALL_TESTS to return 1, it is appropriate to write it there.